mitodl/ocw-studio
# fix file path formatting issue in test course data export

# What are the relevant tickets?
Closes https://github.com/mitodl/ocw-studio/issues/2057

# Description (What does it do?)
This PR fixes an issue with `ExportWebsiteContentSerializer`. Currently when you run content through it, the `file` property on the objects contains a fully qualified URL to the content in S3. File paths in exports should be relative. The serializer was altered to keep only the relative path to the file, and the checked in fixtures were updated.

# How can this be tested?
 - Run `docker compose exec web ./manage.py export_test_sites`
 - Verify that the `file` paths in `test_website_content.json` do not contain fully qualified URLs. Check the changed files in this PR to see an example of what they used to be in `test_site_fixtures/test_website_content.json`
